<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>麻将连连看</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>1600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 91, 51);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>50</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="MyLabel" name="label_reset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/new/prefix1/pic/reset.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_tips">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/new/prefix1/pic/tips.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_auto">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/new/prefix1/pic/auto.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
